packages/@lwc/engine/src/framework/restrictions.ts

```diff
@@ -402,7 +400,7 @@ function getLightningElementPrototypeRestrictionsDescriptors(proto: object): Pro
                 } else if (attribute) {
                     msg.push(`Instead access it via \`this.getAttribute("${attribute}")\`.`);
                 }
-                logError(msg.join('\n'), getComponentVM(this).elm);
+                logError(msg.join('\n'));
```

Comment:
any particular reason for not displaying the error stack?

Reply:
Fixed, I forgot to add it back.